### PR TITLE
Copy Sync: Infinite Scroll / Jetpack mobile theme

### LINKED ISSUE #34235: Copy Sync: Infinite Scroll / Jetpack mobile theme
Before:

Screenshot 2019-06-24 at 17 48 49

After:

See PR https://github.com/Automattic/jetpack/pull/12815

As part of a [recent copy sprint](https://github.com/Automattic/jetpack/projects/23) on Jetpack, many small changes were made to the settings and interface copy. This issue is a sync of the original Jetpack issue, it can be found here:

https://github.com/Automattic/jetpack/issues/9734

### PR DESCRIPTION
**Pull request by scottsweb**
#### Changes proposed in this Pull Request

* Brings copy for the theme enhancements setting panel in sync with Jetpack. These particular changes will roll out with Jetpack 7.6

#### Testing instructions

* Goto settings -> writing -> theme enhancements
* Verify copy changes

Before:

60036981-5ff3dc00-96a8-11e9-8f99-a9f4b2207c90

After:

Screenshot 2019-07-02 at 16 22 09

Jetpack:

59934823-50745900-9444-11e9-9310-ed40b4b379b2

Fixes #34235

### CODE REVIEW

**Comment by scottsweb** [comment]
Jetpack PR: https://github.com/Automattic/jetpack/pull/12815

File: client/my-sites/site-settings/theme-enhancements.jsx [hunk 1]
```diff
@@ -178,7 +183,13 @@ class ThemeEnhancements extends Component {
 					) }
 					link="https://jetpack.com/support/mobile-theme/"
 				/>
-
+				<FormLegend>{ translate( ' Mobile Theme' ) }</FormLegend>
```
Comment (a8ci18n) [inline client/my-sites/site-settings/theme-enhancements.jsx]: Hi! I've found a possible matching string that has already been translated **50** times:
`translate( 'Mobile Theme' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/34235-theme-enhancements)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/site-settings/theme-enhancements.jsx [hunk 2]
```diff
@@ -195,7 +206,7 @@ class ThemeEnhancements extends Component {
 					{ this.renderToggle(
 						'wp_mobile_featured_images',
 						! minilevenModuleActive,
-						translate( 'Hide all featured images' )
+						translate( 'Show featured images' )
```
Comment (davemart-in) [inline client/my-sites/site-settings/theme-enhancements.jsx]: Just double checking that the change from Hide all to Show was intentional.

Comment (scottsweb) [reply client/my-sites/site-settings/theme-enhancements.jsx]: Yes! Thank you for checking. We have had this incorrectly labelled for a long time. Jetpack looks like this:

Screenshot 2019-07-17 at 13 29 35

**Pull request merged**